app/main.py

```diff
@@ -1,13 +1,13 @@
-# from models import Actor
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
-#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
-#     print(Actor.objects.all())
-#     Actor.objects.update(2, "Daniel", "Radcliffe")
-#     print(Actor.objects.all())
-#     Actor.objects.delete(1)
-#     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
```

Comment:
The `ActorManager` instance is created and assigned to `Actor.objects`. Ensure that the database connection is closed when the application terminates to prevent resource leaks. Consider implementing a method in `ActorManager` to close the connection and call it at the end of the script or use a context manager.